ansible/roles/operator-pipeline/templates/openshift/pipelines/community-hosted-pipeline.yml

```diff
@@ -125,10 +125,34 @@ spec:
         - name: remove-matching-namespace-labels
           value: "true"
 
+    # Set a pending GitHub status check first to provide the user with immediate feedback.
+    - name: set-github-status-pending
+      taskRef:
+        name: set-github-status
+      runAfter:
+        - set-github-started-label
+      params:
+        - name: pipeline_image
+          value: "$(params.pipeline_image)"
+        - name: git_repo_url
+          value: $(params.git_repo_url)
+        - name: commit_sha
+          value: $(params.git_commit)
+        - name: description
+          value: "Pipeline for operator tests has started."
+        - name: state
```

Comment:
When looking into the code, I see a parameter named `status` instead of `state`. Is this accounted for?